src/onboarding.md

```diff
@@ -0,0 +1,53 @@
+# Onboarding to the Project
+
+This document is a starting point for a new team member (or a refresh for
+existing team members) on things that are useful to know as a member of the
+project, in particular in terms of where to raise concerns or get help.
+
+## Relationship to Council
+
+The Leadership Council formally takes positions on behalf of the Project when
+needed. All Rust project members are officially represented by approximately
+one member of the council (some teams parent up to two or more members).
+
+Generally speaking, any concerns you have that are either more widely
+applicable (i.e., not just your team) or you feel aren't being fully handled
+within your team can be escalated to the Council. Note that interpersonal
+concerns and/or code of conduct violations should always be directed to the
+Moderation team.
+
+Escalating to the Council can be done via:
+
+- Zulip, on [#council](https://rust-lang.zulipchat.com/#narrow/channel/392734-council)
```

Comment:
I wonder whether we should mention filing an issue on the `leadership-council` repository and nominating it.  While raising the matter on Zulip may generate some discussion, a nominated issue is how something formally makes its way into our inbox.